presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -531,6 +531,10 @@ qualifiedName
     : identifier ('.' identifier)*
     ;
 
+tableVersionExpression
+    : FOR tableVersionType=(TIMESTAMP | VERSION) AS OF valueExpression          #tableVersion
```

Comment:
Thank you for this very insightful comment.  I do indeed see this in the SQL spec in section 4.14 (Periods), which I didn't come across until now.  I see no reason why not to follow the spec.

For `VERSION AS OF`, however, I don't see a corresponding entry in the spec.  I am wondering if we can introduce a function to help with this: translate a version to a date, which can then be used with the single spec-compliant `SYSTEM_TIME AS OF` syntax.  Thoughts?

CC: @aditi-pandit @elharo @kaikalur @rschlussel since you are subscribed to spec modifications.  I would like to hear your thoughts on this.

Reply:
I have created a new issue for keywords in the time travel syntax https://github.com/prestodb/presto/issues/21396

Reply:
I have added SYSTEM_TIME AND SYSTEM_VERSION as the alias names based on feedback.